components/Impact/Carousel.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+
+import { blogs } from '../../data/impact';
+import styles from '../../styles/pages/impact.module.scss';
+
+import BlogCard from './BlogCard';
+
+function Carousel() {
+    const [ activeItem, changeActiveItem ] = useState(0); // keeps track of blog index (0, 1, or 2) that is currently active
+    const blogInfo = blogs[activeItem]; // stores blog info for the blog number that is currently active
+
+    const handleNext = () => {
+        if (activeItem >= (blogs.length - 1))
+            changeActiveItem(0);
+        else
+            changeActiveItem(activeItem+1);
```

Comment:
Though it uses slightly more lines (and looks a bit uglier), I'd recommend wrapping this logic in curly braces. JS is quite loose with syntax which makes it easy to write, but to make this code more robust to future changes, we should explicitly lay out how the logic flows using these braces.

```suggestion
        if (activeItem >= (blogs.length - 1)) {
            changeActiveItem(0);
        }
        else {
            changeActiveItem(activeItem+1);
        }
```